kms/src/main/java/com/example/Asymmetric.java

```diff
@@ -114,20 +115,18 @@ public static String encryptRSA(String message, CloudKMS client, String keyPath)
 
     Cipher cipher = Cipher.getInstance("RSA/NONE/OAEPWITHSHA256ANDMGF1PADDING", "BC");
     cipher.init(Cipher.ENCRYPT_MODE, rsaKey);
-    byte[] ciphertext = cipher.doFinal(message.getBytes(StandardCharsets.UTF_8));
-    return Base64.getEncoder().encodeToString(ciphertext);
+    return cipher.doFinal(plaintext);
   }
   // [END kms_encrypt_rsa]
 
   // [START kms_sign_asymmetric]
   /** Create a signature for a message using a private key stored on Cloud KMS */
-  public static String signAsymmetric(String message, CloudKMS client, String keyPath)
+  public static byte[] signAsymmetric(byte[] message, CloudKMS client, String keyPath)
```

Comment:
I think message is better in this context, since that's the term typically used in cryptography to describe the data being signed: https://en.wikipedia.org/wiki/Digital_signature